pkg/test/component_sns.go

```diff
@@ -85,9 +86,16 @@ func doRunSns(name string, configMap configInput) {
 		},
 		HealthCheck: func() error {
 			snsClient := ProvideSnsClient(name)
-			_, err := snsClient.ListTopics(&sns.ListTopicsInput{})
 
-			return err
+			_, err = snsClient.ListTopics(&sns.ListTopicsInput{})
```

Comment:
Did you take a look at the `ListTopicsResponse`?
Maybe it contains something interesting like a "PleaseRetryAgainIn1Second" response.

Reply:
nothing interesting there that we could make use of